src/main/java/uk/gov/hmcts/reform/blobrouter/services/storage/BlobContainerClientProxy.java

```diff
@@ -50,13 +56,20 @@ public void upload(
         TargetStorageAccount targetStorageAccount
     ) {
         try {
-            get(targetStorageAccount, destinationContainer)
-                .getBlobClient(blobName)
-                .getBlockBlobClient()
+            final BlockBlobClient blockBlobClient =
+                get(targetStorageAccount, destinationContainer)
+                    .getBlobClient(blobName)
+                    .getBlockBlobClient();
+
+            logger.info("Uploading blob content to Container: {}", destinationContainer);
```

Comment:
Worth adding blob name to this log entry, as well.

Reply:
And to the 'finished' entry.

Reply:
Have done